### ISSUE TITLE
Add javac-wrapper.sh to release archive [phab:D242]

### ISSUE THREAD

**Issue by kythe** [google/kythe#458]
_Created by **schroederc** at 2015-05-12 21:18:09_:

**Comment by kythe** [comment]
_**fromberger** commented inline (context lost during migration) at 2015-05-13 21:36:16_:

I'd like to vote for some defensive argument checking, as the diagnostics on a bogus recursive `chown` can be kind of opaque.
```
local path="${1:?missing path}"
chown -R "$(stat '$path' -c $u:%g)" "$path"
```

**Comment by kythe** [comment]
_**fromberger** wrote at 2015-05-13 21:36:16_:

Just one small thing.

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-05-13 21:39:53_:

review changes

**Comment by kythe** [comment]
_**schroederc** commented inline (context lost during migration) at 2015-05-13 21:40:17_:

Done

**Comment by kythe** [comment]
_**fromberger** accepted this revision at 2015-05-13 21:43:46_

**Comment by kythe** [comment]
_**schroederc** closed via commit 63a4aeff91767c902cca2fad0670835f0a6c52c1 at 2015-05-13 21:44:25_
